style.css

```diff
@@ -0,0 +1,48 @@
+body{
+  background-color: #eee;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Helvetica Neue', Arial, sans-serif;
```

Comment:
please do remove the unused code as comment because it makes your code not to look clean